api/python/provisioner/errors.py

```diff
@@ -115,13 +115,11 @@ class SWUpdateRepoSourceError(ProvisionerError, ValueError):
 
     def __init__(self, source: str, reason: str):
         self.source = source
-        self.reason = reason
+        self.reason = (f'repo source "{self.source}" is not acceptable, '
```

Comment:
why do we need that change: initially that attribute kept original reason for later use as per consumer needs. I don't think we should construct a string ready for __str__, it would limit the usage of the attribute.

Reply:
I did that for proper error message understanding on console output. Returned code back for now.